### PR TITLE
Add ability to upload an event icon

### PR DESCRIPTION
**Pull request by akrabat**
Add icon upload to the edit event form.

Also enable configuration of the base URL from which to serve images, so that we can see that it's working in the development environment (if you change you config.php as per the dist file).

This is the web2 part of [JOINDIN-530](https://joindin.jira.com/browse/JOINDIN-530).

Note: only merge this if https://github.com/joindin/joindin-api/pull/289 is accepted! Also, this PR should be updated to remove the `image_base_url` config setting if https://github.com/joindin/joindin-api/pull/292 is accepted.

### CODE REVIEW

File: config/config.php.dist [hunk 1]
```diff
@@ -19,6 +19,7 @@ $config = array(
         'twig' => array(
             // set to the folder name to cache into, or false to disable
             'cache' => false
-        )
+        ),
+        'image_base_url' => 'http://dev.joind.in/inc',
```
Comment (magicmonkey) [inline config/config.php.dist]: Will it actually be getting the full URI from the API response?

Comment (akrabat) [inline config/config.php.dist]: Separate PR for that: https://github.com/joindin/joindin-api/pull/292

Comment (magicmonkey) [inline config/config.php.dist]: Ah, good good, ta.

**Comment by lornajane** [comment]
Replacing this PR with https://github.com/joindin/joindin-web2/pull/336 which extends this one.

**Pull request closed without merging**